### PR TITLE
[AIRFLOW-3884] Fixing doc checker, no warnings allowed anymore and fixed the current…

### PR DESCRIPTION
**Pull request by ffinfo**
… warnings

Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title.
  - https://issues.apache.org/jira/browse/AIRFLOW-3884

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

**Comment by feng-tao** [comment]
the only concern I have is whether we should modify the CHANGELOG file just for solving the doc warning issue. Otherwise LGTM

File: airflow/contrib/executors/mesos_executor.py [hunk 1]
```diff
@@ -49,7 +45,7 @@ def get_framework_name():
 
 # AirflowMesosScheduler, implements Mesos Scheduler interface
 # To schedule airflow jobs on mesos
-class AirflowMesosScheduler(mesos.interface.Scheduler, LoggingMixin):
+class AirflowMesosScheduler(mesos.interface.Scheduler):
```
Comment (feng-tao) [inline airflow/contrib/executors/mesos_executor.py:48]: I think @ashb has asked about deprecating this executor.

Comment (ffinfo) [reply airflow/contrib/executors/mesos_executor.py:48]: ye indeed, but for now this did fix the docs

**Comment by ffinfo** [comment]
I know also din't like that but don't think there is an other way, accepting warnings. Does not really feel like a good idea also

**Review by ashb — APPROVED** [review]

**Comment by ffinfo** [comment]
My change did work but I do see a test failing, something with `pinotdb.db`. I have seen this also on an other branch once I did a sync with master, see also: https://travis-ci.org/apache/airflow/jobs/492321498

**Comment by ffinfo** [comment]
@ashb and @feng-tao Maybe we should merge this even if travis is failing? Then we can fix this pinotdb.db issue in a new PR

**Comment by ashb** [comment]
Yeah I noticed that pinto.db error last night but haven't had a chance to dig in to it yet.

**Comment by feng-tao** [comment]
@ashb , @ffinfo I may know what happens. Let me submit a quick fix.

**Comment by ashb** [comment]
> Pull requests that have a failing status can’t be merged on a phone. 

Grr

**Comment by feng-tao** [comment]
pr (https://github.com/apache/airflow/pull/4704) is created.

**Comment by kaxil** [comment]
Thanks @ffinfo and @feng-tao

**Pull request merged**